libs/domains/organizations/feature/src/lib/hooks/use-repositories/use-repositories.tsx

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query'
+import { type GitProviderEnum } from 'qovery-typescript-axios'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseRepositoriesProps {
+  organizationId: string
+  gitProvider: GitProviderEnum | string
+  gitToken?: string
+  enabled?: boolean
+}
+
+export function useRepositories({ organizationId, gitProvider, enabled, gitToken }: UseRepositoriesProps) {
+  return useQuery({
+    ...queries.organizations.repositories(organizationId, gitProvider, gitToken),
+    meta: {
+      notifyOnError: true,
+    },
+    enabled: enabled,
+    refetchOnWindowFocus: false,
```

Comment:
This should not be added to all `useQuery`. This was needed only for WS because it's not a simple HTTP query

Reply:
I added it for these queries (auth provider, repository, branches), because when we come back on the browser we have an unnecessary reload with the spinner displayed. I think is not necessary because the data has not changed a lot from the API